addons/apecloud-mysql/templates/_helpers.tpl

```diff
@@ -167,7 +167,16 @@ lifecycleActions:
         - /bin/sh
         - -c
         - |
-          /tools/syncerctl switchover --primary "$KB_LEADER_POD_NAME" ${KB_SWITCHOVER_CANDIDATE_NAME:+--candidate "$KB_SWITCHOVER_CANDIDATE_NAME"}
+          if [ -z "$KB_SWITCHOVER_ROLE" ]; then
+              echo "role can't be empty"
+              exit 1
```

Comment:
The KB_SWITCHOVER_ROLE might be empty by design, but a non-zero exit indicates that the switchover action execution failed.

Reply:
I think in most cases we can rely on kb's role info. Engines who need more precise role info can later change to a way that gets role info directly from kernel.